src/python/gudhi/wasserstein.py

```diff
@@ -0,0 +1,75 @@
+import numpy as np
+import scipy.spatial.distance as sc
+try:
+    import ot
+except ImportError:
+    print("POT (Python Optimal Transport) package is not installed. Try to run $ pip install POT")
```

Comment:
@VincentRouvreau should we also mention "conda install -c conda-forge pot"?

Reply:
I added this suggestion (and left the pip one, just in case.).